linux_os/guide/services/mail/package_sendmail_removed/rule.yml

```diff
@@ -32,6 +32,7 @@ references:
     iso27001-2013: A.12.1.2,A.12.5.1,A.12.6.2,A.14.2.2,A.14.2.3,A.14.2.4,A.9.1.2
     cis-csc: 11,14,3,9
     anssi: NT28(R1)
+    srg: SRG-OS-000095-GPOS-00049
```

Comment:
Moved to SRG-OS-000480-GPOS-00227 to ensure that the package is not installed.  I think that if a rule is in the OSPP profile, then we should map to something in the SRG otherwise the two bodies of content will not be the same.

Reply:
Because this rule is enabled in OSPP, keeping it enabled here as well.

Created https://github.com/ComplianceAsCode/content/issues/5013 to begin discussion of why sendmail is being evaluated.